src/main/java/carRacingGame/Car.java

```diff
@@ -0,0 +1,41 @@
+package carRacingGame;
+
+public class Car {
+	public static final int INITIAL_POSITION = 0;
+	public static final String SPACE_BETWEEN_NAME_DISTANCE = " : ";
+	public static final String DISTANCE_OUTPUT = "-";
+	private final String name;
+	int position = INITIAL_POSITION;
```

Comment:
```suggestion
	public static final int INITIAL_POSITION = 0;
	public static final String SPACE_BETWEEN_NAME_DISTANCE = " : ";
	public static final String DISTANCE_OUTPUT = "-";
	
	private final String name;
	private int position = INITIAL_POSITION;
```

상수와 변수 사이에는 공백을 추가하여 구분해주면 편하게 읽을 수 있어요. 

추가로 position에 접근 제어자가 빠졌네요 😀

다음 단계에서 반영 부탁드려요!